Automattic/wp-calypso
# Update/add earn banner to customer home

#### Changes proposed in this Pull Request

This PR adds Earn cards to both the Primary and Secondary sections of My Home. The API will only show these cards to English users (I'll remove that restriction once the translations are in).

The secondary Earn card appears at the top of the Learn and Grow section of the TASK_EARN_FEATURES view.

Here's a screenshot:
<img width="1107" alt="Screen Shot 2020-05-29 at 12 16 12 PM" src="https://user-images.githubusercontent.com/35781181/83281638-62a75d00-a1a6-11ea-869e-d58b06dd9518.png">

API updates to show these cards are in D44038-code.

#### Testing instructions
* Checkout branch (git fetch origin && git checkout -b update/add-earn-banner-to-customer-home origin/update/add-earn-banner-to-customer-home).
* Patch your sandbox with D44038-code.
* Edit your host file to point API requests to your sandbox.
* [Confirm](https://public-api.wordpress.com/?amisandboxed) that your API requests are sandboxed.
* Run Calypso.
* Create a new site and confirm that the Earn card appears after the site creation tasks and that it's dismissable.
* Check the links in both Earn cards and confirm they go to the right areas.
* Load the Earn view in [dev mode](http://calypso.localhost:3000/home/:siteslug?dev=true&view=VIEW_EARN_FEATURES&flags=-home/layout-dev) to verify all links.
